docs/assignment1.md

```diff
@@ -1,62 +1,58 @@
 
-Be creative here!
-
-Don’t forget to use links to external URLs (e.g., the direct link to the Fantasy soccer league you are getting inspiration from, the link to the original video game you are getting inspiration from, etc.), if applicable. 
-
-Maximum number of words for this section: 1000
 
 ## Features
-Author(s): `name of the team member(s) responsible for this section`
-
-<When defining both functional features and quality requirements, remember that you will need to come back to them in Assignments 2 and 3 and explicitly state how specific parts of models/implementation satisfy them.>
+Author(s): Beth, Sarah, Ben, Sam, Gemma
 
 ### Functional features
 
-As a preamble to the table, you can discuss the main line of reasoning you applied for defining the functional features provided in the table.
+In order to make the game as generic as possible, we want to include Functional Features that will be used in the game such as Commands, Movements, etc. that the player will be able to use during the game. 
 
 | ID  | Short name  | Description  |
 |---|---|---|
-| F1  | Tags | Code snippets can be tagged via freely-defined labels called tags  |
-| F2  | Commands  | The player can control the main character by issuing command-line commands following this syntax: `command-name [target-objects]*`. The available `command-names` are the following: <br/> - move: ... <br/> - use: ... <br/> - inspect: ... <br/> |
-| F3  | Movements  | The main character shall move freely in the environment according  |
-| F4  | ... | ... |
+| F1  | Command | Player can control the main character using command line commands with the following syntax: ‘[command] [object]’. The commands we will implement are: move, open, break, inspect, help   |
+| F2  | Movements  | Player can move through the environment using command: ‘move to [location]’  |
+| F3  | Help Messages  | After a certain amount of time, user will be given hints of what actions to perform or movements to make  |
+| F4  | Objects | User can use commands on objects (e.g. bikes or metro in Amsterdam, coffee, doors) in order to progress. Some objects will be used to perform actions and some will be used as an energy booster to get through the game.  |
```

Comment:
You are classifying your objects into two categories: 1)Actionable objects that seem to be mainly there to enhance movement and transportation. 2) energy boosters. Those could have been mentioned separately into two features.